keyboards/handwired/prime_exl/info.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
This file is not valid for QMK purposes. Looking it over it appears to be a mix of QMK Configurator and KBFirmware data.

If you're trying to add support for this keyboard in the QMK Configurator, I can help with that.

Reply:
done.

Reply:
I believe I have updated the JSON as requested. though I'm not sure of a way to test.

Reply:
For testing the visual mock-up: open the Configurator, hold `Ctrl`, then hold `Shift`, then tap `I`, and upload the file. This only tests that the mock-up is drawn properly; it doesn't test functionality.